### PR TITLE
changed user and added user role enum and employee model

### PR DESCRIPTION
**Pull request by beatakoziel**

### CODE REVIEW

File: Slicesy.PizzaOrder.WebApi/Repositories/Models/Enums/UserRole.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace Slicesy.PizzaOrder.WebApi.Repositories.Models.Enums
+{
+    public enum UserRole
+    {
+        Employee, PizzeriaAdmin, CompanyAdmin
```
Comment (Haseoo) [inline Slicesy.PizzaOrder.WebApi/Repositories/Models/Enums/UserRole.cs]: Jeszcze Admin

**Review by Haseoo — CHANGES_REQUESTED** [review]

File: Slicesy.PizzaOrder.WebApi/Repositories/Models/Enums/UserRole.cs (hunk 1 above)
Comment (beatakoziel) [reply Slicesy.PizzaOrder.WebApi/Repositories/Models/Enums/UserRole.cs]: XDDDD

File: Slicesy.PizzaOrder.WebApi/Repositories/Models/User.cs [hunk 2]
```diff
@@ -24,8 +25,13 @@ public User(string login, string password, long? permittedResourceId = null)
         [Required(ErrorMessage = "Password is required")]
         public string Password { get; set; }
 
+        [Required(ErrorMessage = "Active info is required")]
         public bool Active { get; set; }
 
+        [Required(ErrorMessage = "Pizzeria or company id is required")]
         public long? PermittedResourceId { get; set; }
```
Comment (Haseoo) [inline Slicesy.PizzaOrder.WebApi/Repositories/Models/User.cs:31]: Bez tego sprawdzania

File: Slicesy.PizzaOrder.WebApi/Repositories/Models/User.cs [hunk 3]
```diff
@@ -24,8 +25,13 @@ public User(string login, string password, long? permittedResourceId = null)
...
+
+        [Required(ErrorMessage = "Role is required")]
+        public UserRole Role { get; set; }
```
Comment (Haseoo) [inline Slicesy.PizzaOrder.WebApi/Repositories/Models/User.cs]: Virtual

**Review by Haseoo — CHANGES_REQUESTED** [review]

**Review by Haseoo — APPROVED** [review]

**Pull request merged**